scripts/common/settings.js

```diff
@@ -0,0 +1,68 @@
+const SETTINGS_TABS = {
```

Comment:
Missing `'use strict';`
```suggestion
'use strict';

const SETTINGS_TABS = {
```

Reply:
Fixed